src/main/java/core/basesyntax/dao/impl/UserDaoImpl.java

```diff
@@ -12,21 +15,63 @@ public UserDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public User create(User entity) {
-        return null;
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
+            transaction.commit();
+            return entity;
+        } catch (RuntimeException e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't add movie entity " + entity, e);
```

Comment:
in all other methods also
```suggestion
            throw new RuntimeException("Can't add user entity " + entity, e);
```